subsys/bluetooth/host/monitor.h

```diff
@@ -24,6 +24,8 @@
 #define BT_MONITOR_VENDOR_DIAG	11
 #define BT_MONITOR_SYSTEM_NOTE	12
 #define BT_MONITOR_USER_LOGGING	13
+#define BT_MONITOR_ISO_TX_PKT	18
```

Comment:
Line up with those above?

Reply:
This file is using tab to align these, I will convert it to spaces since github don't seem to be able to format tabs properly.